client/src/components/Form.js

```diff
@@ -41,12 +41,13 @@ class Form extends Component {
 
     const data = { ...this.state.data };
     data[input.name] = input.value;
+    console.log(data)
```

Comment:
Don't forget to remove. Likely there is an eslint rule complaining about this as well:)